contracts-bedrock/helpers/utils.ts

```diff
@@ -1,19 +1,7 @@
-import * as RLP from '@ethersproject/rlp'
-import { BigNumber, BigNumberish } from '@ethersproject/bignumber'
-import { getAddress } from '@ethersproject/address'
-import {
-  hexConcat,
-  stripZeros,
-  zeroPad,
-  arrayify,
-  BytesLike,
-} from '@ethersproject/bytes'
-import { keccak256 } from '@ethersproject/keccak256'
-import { Zero } from '@ethersproject/constants'
-import { ContractReceipt, Event } from '@ethersproject/contracts'
+import { BigNumber, BigNumberish, BytesLike, ContractReceipt, ethers, Event } from 'ethers'
```

Comment:
Using these from ethers directly reduces the number of dependencies, which makes it easier to update dependencies if we ever need to do this again in the future.